packages/metadata-crawler/src/crawler.ts

```diff
@@ -0,0 +1,114 @@
+import { IdentityMetadataWrapper } from '@celo/contractkit'
```

Comment:
Do we already have a helm-chart for this/Added this to the blockscout helm chart?

Reply:
I think the crawler can be executed as a cron. I suggest to do that as part of a different user story. What do you think?